server/routes/accessDeniedController.ts

```diff
@@ -0,0 +1,16 @@
+import { RequestHandler } from 'express'
+import paths from '../constants/paths'
+
+export default class AccessDeniedController {
+  public constructor() {}
+
+  public view(): RequestHandler {
+    return async (req, res) => {
+      if (req.session.booker) {
+        return res.redirect(paths.HOME)
+      }
+
+      return res.render('accessDenied')
```

Comment:
I feel I'm getting slightly obsessed by ternaries but nothing wrong with this at all 😄 

`return req.session.booker ? res.redirect(paths.HOME) : res.render('accessDenied')`

Reply:
Good suggestion; looks neater now. Have updated!